### PR TITLE
Minor cleanup.

### PR DESCRIPTION
**Pull request by Random-Liu**
Addresses:
* https://github.com/kubernetes-incubator/cri-containerd/pull/479#discussion_r155183127
* https://github.com/kubernetes-incubator/cri-containerd/pull/470#discussion_r156522507
* https://github.com/kubernetes-incubator/cri-containerd/pull/470#discussion_r156522507

/cc @mikebrow @yanxuean 
Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: pkg/server/sandbox_status_test.go [hunk 1]
```diff
@@ -78,7 +77,24 @@ func TestPodSandboxStatus(t *testing.T) {
...
+		}
+		expected.State = state
+		got := toCRISandboxStatus(sandbox.Metadata, status, createdAt, ip)
+		assert.Equal(t, expected, got)
+	}
+}
 
-	got := toCRISandboxStatus(sandbox.Metadata, state, createdAt, ip)
-	assert.Equal(t, expected, got)
+func TestPodSandboxState(t *testing.T) {
```
Comment (yanxuean) [inline pkg/server/sandbox_status_test.go]: Empty function?

Comment (Random-Liu) [reply pkg/server/sandbox_status_test.go]: Oops, forgot removing.

**Comment by Random-Liu** [comment]
Apply LGTM based on https://github.com/kubernetes-incubator/cri-containerd/pull/493#pullrequestreview-83047737

**Pull request merged**